ui/app/src/views/dashboard/Panel.tsx

```diff
@@ -29,7 +29,8 @@ export interface PanelProps extends CardProps {
 function Panel(props: PanelProps) {
   const { definition, ...others } = props;
   const [contentElement, setContentElement] = useState<HTMLDivElement | null>(null);
-
+  const showPanelHeader = definition.display.show_panel_header ?? true;
```

Comment:
Went with this so StatChart's backgroundColor could extend to the edge of the panel without any whitespace (and so display.name isn't duplicated), thoughts on a better name or approach?

Reply:
As discussed on call, planning to remove this and instead use theme to customize.

Reply:
👍